homework_4/task_4.py

```diff
@@ -0,0 +1,6 @@
+import re
+
+name = ["FirstItem", "FriendsList", "MyTuple"]
+name = str(name)
+name = re.sub(r'(?<!^)(?=[A-Z])', '_', name).lower()
+print(name)
```

Comment:
![image](https://user-images.githubusercontent.com/25909810/221915314-32fec184-db6e-47c0-b09a-3796b9c031db.png)
why do you have _ at the beginning of every element?